<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1448</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#2ea890;&quot;&gt;USER NAME&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_name">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#2ea890;&quot;&gt;PASSWORD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dxu">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ĐỀ XUẤT USER NAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dxp">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ĐỀ XUẤT PASSWORD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dk">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ĐĂNG KÍ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#2ea890;&quot;&gt;EMAIL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ql">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>QUAY LẠI TRANG TRƯỚC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
